TestModels/SimpleTypes/SimpleBoolean/runtimes/rust/src/client.rs

```diff
@@ -1,3 +1,6 @@
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+// Do not modify this file. This file is machine generated, and any changes to it will be overwritten.
 // Code generated by software.amazon.smithy.rust.codegen.smithy-rs. DO NOT EDIT.
```

Comment:
This turned out to not be true (for now), so we should remove it. The line above already marks it as generated after all.